app/backend/prisma/schema.prisma

```diff
@@ -115,6 +115,7 @@ model User {
   email         String?   @unique
   emailVerified DateTime?
   image         String?
+  capacityHours Float     @default(0)
```

Comment:
I think you are in the wrong table/model.
This should be handled in the `TeamMembership` table.

And we currently use for all durations minutes in `Int`s.

@moncapitaine, what do you think?
Maybe it also makes sense to make this field nullable, as this might be an optional feature.